packages/angular_devkit/build_angular/src/builders/ssr-dev-server/specs/proxy_spec.ts

```diff
@@ -106,4 +106,49 @@ describe('Serve SSR Builder', () => {
       await new Promise<void>((resolve) => proxyServer.close(() => resolve()));
     }
   });
+
+  it('proxies requests based on the proxy configuration ts file provided in the option', async () => {
+    const proxyServer = http.createServer((request, response) => {
+      if (request.url?.endsWith('/test')) {
+        response.writeHead(200);
+        response.end('TEST_API_RETURN');
+      } else {
+        response.writeHead(404);
+        response.end();
+      }
+    });
+
+    try {
+      await new Promise<void>((resolve) => proxyServer.listen(0, '127.0.0.1', resolve));
+      const proxyAddress = proxyServer.address() as import('net').AddressInfo;
+
+      host.writeMultipleFiles({
+        'proxy.config.ts': `
```

Comment:
TypeScript files are not supported. In this case it works because the contents are plain JavaScript.

Reply:
You are right, I'll change it to js file.